motr/m0crate/tests/test1_io.yaml

```diff
@@ -37,6 +37,7 @@ LOG_LEVEL: 4  # err(0), warn(1), info(2), trace(3), debug(4)
 WORKLOAD_SPEC:               # Workload specification section
    WORKLOAD:                 # First Workload
       WORKLOAD_TYPE: 1       # Index(0), IO(1)
+      #POOL_FID: <0x6f00000000000001:0x2f> # The fid of the pool
```

Comment:
Repeating the parameter name in the comment is not of a big value. I'd suggest adding something more useful instead, for example: "Default pool is used, if not set".

Reply:
Done.